src/components/Events.vue

```diff
@@ -39,6 +40,12 @@ export default {
 				return [];
 			},
 		},
+		isEmpty: {
+			type: Boolean,
+			default() {
+				return false;
+			},
+		},
 	},
 	computed: {
 		hasEvents() {
```

Comment:
add `shouldLoadingEvents` here

```suggestion
    shouldLoadEvents() { 
        return ...
    },